src/MagnumPlugins/DrFlacAudioImporter/Test/DrFlacImporterTest.cpp

```diff
@@ -42,30 +42,83 @@ class DrFlacImporterTest: public TestSuite::Tester {
 
         void mono8();
         void mono16();
+        void mono24();
+
+        void stereo8();
         void stereo16();
+        void stereo24();
+
+        void quad16();
+        void quad24();
+
+        void surround51Channel16();
+        void surround51Channel24();
 };
 
 DrFlacImporterTest::DrFlacImporterTest() {
     addTests({
               &DrFlacImporterTest::mono8,
               &DrFlacImporterTest::mono16,
-              &DrFlacImporterTest::stereo16});
+//              &DrFlacImporterTest::mono24,
+
+              &DrFlacImporterTest::stereo8,
+              &DrFlacImporterTest::stereo16,
+//              &DrFlacImporterTest::stereo24,
+
+              &DrFlacImporterTest::quad16,
+//              &DrFlacImporterTest::quad24,
+
+              &DrFlacImporterTest::surround51Channel16
+//              &DrFlacImporterTest::surround51Channel24
```

Comment:
@mosra These are commented out because I intend to enable them as soon as I have 24 bit audio working.